packages/grafana-ui/src/components/MatchersUI/FieldValueMatcher.tsx

```diff
@@ -62,7 +62,7 @@ export const FieldValueMatcherEditor = ({ options, onChange }: Props) => {
   );
 
   const opts = options ?? {};
-  const isBool = isBooleanReducer(options.reducer);
+  const isBool = isBooleanReducer(options?.reducer);
```

Comment:
the rest of the code uses `opts` from the line above, so let's change this to also use `opts.reducer`?